testfiles/5ZoneDetailedIceStorageCubicLinear.idf

```diff
@@ -3058,9 +3058,9 @@
     0.5,                     !- Capacity {GJ}
     Ice Tank Inlet Node,     !- Inlet Node Name
     Ice Tank Outlet Node,    !- Outlet Node Name
-    CubicLinear,             !- Discharging Curve Object Type
+    LMTDMassFlow,            !- Discharging Curve Object Type
     DischargeCurve,          !- Discharging Curve Name
-    CubicLinear,             !- Charging Curve Object Type
+    LMTDMassFlow,            !- Charging Curve Object Type
```

Comment:
Need new field names here and in some of the other idfs.

Reply:
@mjwitte I think I got all of them now.  Two were already done.  Missed the others in the previous pass.  Good catch, again.